src/modules/add-site/components/blueprint-deeplink.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+	Icon,
+} from '@wordpress/components';
+import { check } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { Blueprint } from 'src/stores/wpcom-api';
+
+interface BlueprintDeeplinkProps {
+	selectedBlueprint?: Blueprint;
+}
+
+export default function BlueprintDeeplink( { selectedBlueprint }: BlueprintDeeplinkProps ) {
+	const { __ } = useI18n();
+
+	const blueprintTitle = selectedBlueprint?.title || __( 'Blueprint' );
+	const blueprintDescription = selectedBlueprint?.excerpt || '';
+
+	return (
+		<VStack className="text-center w-full" alignment="top" spacing={ 0 }>
+			<Heading className="text-center text-[32px] text-gray-900 mb-[59px]" weight={ 500 }>
+				{ __( 'Start from a Blueprint' ) }
```

Comment:
```suggestion
				{ __( 'Start with Blueprint' ) }
```
This sounds more grammatically correct to me as Blueprint is feature. Let me know what do you think.

Reply:
> This sounds more grammatically correct to me as Blueprint is feature. Let me know what do you think.

Thanks for the suggestion. The reason I picked `Start from a Blueprint` is because we already use it for the standard Blueprint flow:

<img width="2736" height="1760" alt="CleanShot 2025-11-26 at 10 51 17@2x" src="https://github.com/user-attachments/assets/eb51e6cc-9fbb-48ce-931d-a107741f6da5" />

Reply:
I agree. However, I feel this action is a bit different than the previous one. I would still choose "Start with a Blueprint" over "Start from a Blueprint." I think it's much clearer. However, I leave the final decision to you.

Reply:
I think it makes sense to use `Start from a Blueprint` as the flow is very similar, only the way we get the Blueprint is different. 🙂

If we wanted to differentiate, I could also imagine `Open Blueprint` or `Import Blueprint` as the headings for this screen.

Reply:
Thank you for your feedback Rahul and Gergely. I will leave the title as-is for now, but we can indeed revisit it later, e.g. once we get the design input on this step from Marina.